cmake/options.cmake

```diff
@@ -17,6 +17,22 @@ option(MATIO_SHARED "Build shared matio library, disable for static library" ON)
 # Option to enable position-independent code (PIC)
 option(MATIO_PIC "Enable position-independent code (PIC), i.e., compilation with the -fPIC flag" ON)
 
+# Option to link the MSVC runtime library statically
+option(MATIO_STATIC_LINK_CRT "Link the MSVC runtime library statically" OFF)
+
+if(POLICY CMP0091)
```

Comment:
Maybe move the option inside the policy check, so it is not visible in old CMake GUIs.
And also inside the `MSVC` check. Or do you plan to add something similar for other compilers too?